README.md

```diff
@@ -2,21 +2,31 @@
 
 This is a prototype written in Java to disassemble and assemble SPIR-V binary modules. It provides a Java programming framework to allow client Java applications to assemble binary modules and dissasemble binary modules into text. More information on the format of SPIR-V can be found [here](docs/SPIRV.md). To see an example kernel in SPIR-V go [here](docs/EXAMPLE.md).
 
-## How to Build?
 
-Dependencies:
+## Build on Linux
 
-- Java JDK 21. It also supports JDK 17, 11 and 8 but this branch will be deprecated soon. We highly recoomand JDK >= 11. 
-- Maven (>= 3.6.3) 
+Dependencies:
+- Java JDK 21. It also supports JDK 17, 11 and 8 but this branch will be deprecated soon. We highly recommend JDK 21.
+- Maven >= 3.6.3
 
 Clone the repository:
 
 ```bash
-$ git clone https://github.com/beehive-lab/beehive-spirv-toolkit.git
-$ cd spirv-beehive-toolkit
-$ mvn clean install
+git clone https://github.com/beehive-lab/beehive-spirv-toolkit.git
+cd spirv-beehive-toolkit
+mvn clean install
 ```
 
+## Build on Windows
+
+Working configurations:
+- Lenovo IdeaPad Gaming 3 15IHU6
```

Comment:
My guess is that this is the platform that it has been tested, right? Shall we remove it to be consistent with the Linux outline?

Reply:
Right, it's the platform used for testing. Removing for me ok.